<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsInterpolationDialogBase</class>
 <widget class="QDialog" name="QgsInterpolationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>382</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Interpolation plugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mInputGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vector layers</string>
        </property>
        <property name="buddy">
         <cstring>mInputLayerComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="mInputLayerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mInterpolationAttributeLabel">
        <property name="text">
         <string>Interpolation attribute</string>
        </property>
        <property name="buddy">
         <cstring>mInterpolationAttributeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="mInterpolationAttributeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="mUseZCoordCheckBox">
        <property name="text">
         <string>Use z-Coordinate for interpolation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="mAddPushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="mRemovePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTreeWidget" name="mLayersTreeWidget">
        <column>
         <property name="text">
          <string>Vector layer</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Attribute</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="mOutputGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="mInterpolationLabel">
        <property name="text">
         <string>Interpolation method</string>
        </property>
        <property name="buddy">
         <cstring>mInterpolationMethodComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QComboBox" name="mInterpolationMethodComboBox"/>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="mConfigureInterpolationButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="interpolator.qrc">
          <normaloff>:/options.png</normaloff>:/options.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="mNumberOfColumnsLabel">
        <property name="text">
         <string>Number of columns</string>
        </property>
        <property name="buddy">
         <cstring>mNumberOfColumnsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="mNumberOfColumnsSpinBox">
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="mNumberOfRowsLabel">
        <property name="text">
         <string>Number of rows</string>
        </property>
        <property name="buddy">
         <cstring>mNumberOfRowsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QSpinBox" name="mNumberOfRowsSpinBox">
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mCellsizeXLabel">
        <property name="text">
         <string>Cellsize X</string>
        </property>
        <property name="buddy">
         <cstring>mCellsizeXSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="mCellsizeXSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLabel" name="mCellsizeYLabel">
        <property name="text">
         <string>Cellsize Y</string>
        </property>
        <property name="buddy">
         <cstring>mCellSizeYSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="2">
       <widget class="QDoubleSpinBox" name="mCellSizeYSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="7">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="mXMinLabel">
          <property name="text">
           <string>X min</string>
          </property>
          <property name="buddy">
           <cstring>mXMinLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="mXMinLineEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="mXMaxLabel">
          <property name="text">
           <string>X max</string>
          </property>
          <property name="buddy">
           <cstring>mXMaxLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="mXMaxLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="mYMinLabel">
          <property name="text">
           <string>Y min</string>
          </property>
          <property name="buddy">
           <cstring>mYMinLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="mYMinLineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="mYMaxLabel">
          <property name="text">
           <string>Y max</string>
          </property>
          <property name="buddy">
           <cstring>mYMaxLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="mYMaxLineEdit"/>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="mBBoxToCurrentExtent">
          <property name="text">
           <string>Set to current extent</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mOutputFileLabel">
        <property name="text">
         <string>Output file </string>
        </property>
        <property name="buddy">
         <cstring>mOutputFileLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QLineEdit" name="mOutputFileLineEdit"/>
      </item>
      <item row="5" column="6">
       <widget class="QToolButton" name="mOutputFileButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mInputLayerComboBox</tabstop>
  <tabstop>mInterpolationAttributeComboBox</tabstop>
  <tabstop>mUseZCoordCheckBox</tabstop>
  <tabstop>mAddPushButton</tabstop>
  <tabstop>mRemovePushButton</tabstop>
  <tabstop>mLayersTreeWidget</tabstop>
  <tabstop>mInterpolationMethodComboBox</tabstop>
  <tabstop>mConfigureInterpolationButton</tabstop>
  <tabstop>mNumberOfColumnsSpinBox</tabstop>
  <tabstop>mNumberOfRowsSpinBox</tabstop>
  <tabstop>mCellsizeXSpinBox</tabstop>
  <tabstop>mCellSizeYSpinBox</tabstop>
  <tabstop>mXMinLineEdit</tabstop>
  <tabstop>mXMaxLineEdit</tabstop>
  <tabstop>mYMinLineEdit</tabstop>
  <tabstop>mYMaxLineEdit</tabstop>
  <tabstop>mBBoxToCurrentExtent</tabstop>
  <tabstop>mOutputFileLineEdit</tabstop>
  <tabstop>mOutputFileButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="interpolator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsInterpolationDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>723</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsInterpolationDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>723</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
